web-backend/metrics/src/common/github.rs

```diff
@@ -27,7 +27,8 @@ pub async fn get_repository_info(
     debug!("{:?}", octocrab);
 
     octocrab
-        .get("https://api.github.com/app/", None::<&()>)
+        // .get("https://api.github.com/app/", None::<&()>)
+        .get("https://api.github.com/repos/diem/whackadep", None::<&()>)
```

Comment:
?

Reply:
seems my PAT doesn't have access to `/app` endpoint. I also couldn't make my RSA key registered with GitHub to work for this test. Did you specific any app specific token?

Reply:
@mimoo yeah any idea what this was?

```
key_path.push("resources/keys/whackadep.2021-01-25.private-key.pem");
```

Reply:
good thing I didn't push that one to the repo :D 

all in all that code wasn't used yet (it was supposed to be used to check the stargazer count of a repo IIRC) so I'm not sure if it really worked.